lighthouse-core/config/default.json

```diff
@@ -3,6 +3,7 @@
     "passName": "defaultPass",
     "recordNetwork": true,
     "recordTrace": true,
+    "pauseAfterTraceMs": 500,
```

Comment:
the pause is previous to ending the trace, so i'd think we'd call it something like`recordTraceEndDelay`. wdyt?